package.json

```diff
@@ -66,6 +66,9 @@
     "@types/jest": "^26.0.0",
     "@typescript-eslint/eslint-plugin": "^4.1.0",
     "@typescript-eslint/parser": "^4.1.0",
+    "@types/express": "^4.17.8",
```

Comment:
do we need to have it globally? Can we move types to the package itself?

Reply:
Done